actionpack/lib/action_controller/caching.rb

```diff
@@ -46,6 +46,20 @@ def cache_configured?
         end
     end
 
+    # Cache or yield the block. The cache is suppose to never expire.
+    #
+    # <tt>public</tt> By default the Cache-Control header is private, set this to
+    #   +true+ if you want your application to be cachable by other devices (proxy caches).
```

Comment:
By default, HTTP responses are private, cached only on the user's web browser. To allow proxies to cache the response, set `public: true` to indicate that they can serve the cached response to all users.